### PR TITLE
change default 1p blocking policy on release channel

### PR DESCRIPTION
**Pull request by antonok-edm**
Merges changes from https://github.com/brave/brave-variations/pull/148 into production

### CODE REVIEW

**Review by pes10k — APPROVED** [review]
lgtm, but others should approve too before merging, to make sure 100% on relase is allowed here

**Review by pes10k — APPROVED** [review]
lgtm, thanks for the edits!

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
